pkgs/by-name/oa/oakctl/package.nix

```
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  patchelf,
+  libgcc,
+  versionCheckHook,
```

```suggestion
  libgcc,
```

If I'm not mistaken (not on my computer rn), these two attributes are not used anymore.

---

Sure, let's remove them. Thanks for review and help!